api/app/Models/SkillFamily.php

```diff
@@ -42,15 +39,4 @@ public function skills(): BelongsToMany
     {
         return $this->belongsToMany(Skill::class);
     }
-
-
-    public static function scopeTechnical(Builder $query)
-    {
-        return $query->where('category', '=', ApiEnums::SKILL_CATEGORY_TECHNICAL);
-    }
-
-    public static function scopeBehavioural(Builder $query)
```

Comment:
Got one place still referencing these scopes rather than the new ones. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/97c7e195-1593-4e4f-844c-529d39190492)